### ISSUE TITLE
[BUG] [DaisyUI] missing `collapse-close` classname

### ISSUE THREAD

**Issue by rtritto** [francoismassart/eslint-plugin-tailwindcss#358]
**Describe the bug**
Using `collapse-close` classname for collapse component (https://daisyui.com/components/collapse)
```tsx
<div tabindex={0} class="collapse collapse-close collapse-arrow">
  <div class="collapse-title">
    Collapse title
  </div>

  <div class="collapse-content">
    Collapse content
  </div>
</div>
```
and run ESLint, it's not recognized
```console
22:27  warning  Classname 'collapse-close' is not a Tailwind CSS class!  tailwindcss/no-custom-classname
```

**Note**: I don't know if this issue is related to _daisyui_ or _tailwindcss_

**To Reproduce**
1. `git clone https://github.com/rtritto/template-vite-solid-daisyui.git`
2. `cd template-vite-solid-daisyui`
3. `git checkout bug-collapse-eslint-tailwindcss`
4. `yarn`
5. `yarn eslint .`

**Expected behavior**
`tailwindcss/no-custom-classname` rule doesn't apper on eslint output.

**Environment (please complete the following information):**
 - OS: Windows / Linux
 - Softwares + version used:
     - daisyui 4.12.10
     - tailwindcss 3.4.10
     - eslint 8.57.0
     - eslint-plugin-tailwindcss 3.17.4
     - @typescript-eslint/eslint-plugin 8.3.0
     - @typescript-eslint/parser 8.3.0
     - Node 20.16.0

**eslint config file or live demo**
[.eslintrc.yml](https://github.com/rtritto/template-vite-solid/blob/bug-collapse-eslint-tailwindcss/.eslintrc.yml)

**Comment by kachkaev** [comment]
👋 @rtritto! I did not find `collapse-close` in Tailwind docs: https://tailwindcss.com/docs/border-collapse#collapse

I guess that supporting Daisy UI is out of scope for this plugin. You can specify third-party class names via rule settings though:
https://github.com/francoismassart/eslint-plugin-tailwindcss/blob/master/docs/rules/no-custom-classname.md#whitelist-default-

**Comment by rtritto** [comment]
Thanks, I fixed with:

`.eslintrc.yml`
```yml
rules:
  tailwindcss/no-custom-classname: [1, {
    whitelist: ['collapse-close']
  }]
```
